src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Color;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RANDOM_VALUE = 10;
+    private static final int FIGURE_COUNT = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int chooser = random.nextInt(FIGURE_COUNT);
+        Figure figure = switch (chooser) {
+            case 0 -> new Circle(random.nextInt(MAX_RANDOM_VALUE) + 1);
```

Comment:
I suggested to make MAX_RANDOM_VALUE = 10; not 9, to avoid +1, so please remove +1 everywhere